packages/projection-typeorm/src/entity/PoolMetadata.entity.ts

```diff
@@ -8,17 +8,17 @@ import { StakePoolEntity } from './StakePool.entity';
 export class PoolMetadataEntity {
   @PrimaryGeneratedColumn()
   id?: number;
-  @Column()
+  @Column('varchar')
```

Comment:
I am asking to learn; How are these and the other Column improvements related to the PoC? Maybe some serialization needs?

Reply:
This is not bound to the WebSocket PoC, this is related to adding the hot reload feature.
`tsx` doesn't support `emitDecoratorMetadata`, so TypeORM is not able to guess the column type, so I had to make then explicit.

Reply:
My personal preference would be to split the hotreload and the PoC work in an ideal world.